src/main/java/cubicchunks/worldgen/generator/custom/CustomGeneratorSettings.java

```diff
@@ -149,10 +149,13 @@
     public float diamondOreSpawnMaxHeight = -0.75f;
 
     public int lapisLazuliSpawnSize = 7;
-    public int lapisLazuliSpawnTries = 1;
-    public float lapisLazuliSpawnProbability = 256f / 32f / (256f / Cube.SIZE);
-    public float lapisLazuliHeightMean = 0.25f; // actually vanilla closest fit is 15/64 and 7/64
-    public float lapisLazuliHeightStdDeviation = 0.125f;
+    public int lapisLazuliSpawnTries = 1; //1 try per chunk, just like vanilla
+    public float lapisLazuliSpawnProbability = 0.933307775f; //resulted by approximating triangular behaviour with bell curve
+    public float lapisLazuliHeightMean = -1.0f; // -> first belt at height 0
```

Comment:
It may look like a minor nitpick, but this means that players expecting lapis to be most common at y=6 won't find a lot of it there. It should be offset to that the mean value on the first belt is same as vanilla.

Reply:
Changed to -0.75 (so height of 16 blocks) to mirror vanilla spawn position.